### DISCUSSION TITLE
Copyable Action

### DISCUSSION THREAD

**Discussion by misbahansori** [filamentphp/filament#8694] [Ideas]
Filament already has a built-in copy-to-clipboard functionality on the TextColumn and TextEntry. However, this function is not available on filament actions. It would also be nice if this function was in the filament action.

**Comment by saleh-movahedi** [comment]
> Filament already has a built-in copy-to-clipboard functionality on the TextColumn and TextEntry. However, this function is not available on filament actions. It would also be nice if this function was in the filament action.

I also need this feature

**Comment by MACscr** [comment]
I agree. I need this as well.
